core/src/main/java/hudson/PluginManager.java

```diff
@@ -24,6 +24,7 @@
 package hudson;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.security.ACLContext;
```

Comment:
apart from `jenkins.util.SystemProperties` I think you will find they are grouped :-)